packages/gatsby/content/getting-started/install.md

```diff
@@ -32,7 +32,7 @@ cd ~/path/to/project
 
 ```bash
 yarn policies set-version berry # below v1.22
-yarn set version berry          # on v1.22+
+yarn set version latest          # on v1.22+
```

Comment:
Actually, this is incorrect. `latest` on v1 resolves to the latest v1 release. This should probably just be reverted since it's the part about upgrading from v1. Let's hope that people don't mistake it for upgrading when already using v2.

Reply:
@paul-soporan This has been updated.